projects/showcase/cypress/support/maplibre-helper.ts

```diff
@@ -0,0 +1,78 @@
+import { CypressHelper } from "@shellygo/cypress-test-utils";
+import { Assertable, then } from "@shellygo/cypress-test-utils/assertable";
+import pixelmatch from "pixelmatch";
+export class MapLibreAssertable<T> extends Assertable<T> {
```

Comment:
Shouldn't the assertable have its own file?

Reply:
Not sure. I was thinking all Maplibre's extensions to test utils can be here, so we don't have to go back and forth between source files. WDYT?

Reply:
If it was methods I would agree, but I prefer that every class would have its own file.